reolink_aio/typings.py

```diff
@@ -27,6 +30,177 @@
     },
 )
 
+GetTimeDst = TypedDict(
+    "GetTimeDst",
+    {
+        "enable": bool,
+        "offset": int,
+    }
+)
+
+GetTimeDstRule = TypedDict(
+    "GetTimeDstRule",
+    {
+        "Mon": int,
+        "Week": int,
+        "Weekday": int,
+        "Hour": int,
+        "Min": int,
+        "Sec": int,
+    }
+)
+
+GetTime = TypedDict(
+    "GetTime",
+    {
+        "year": int,
+        "mon": int,
+        "day": int,
+        "hour": int,
+        "min": int,
+        "sec": int,
+        "hourFmt": int,
+        "timeFmt": str,
+        "timeZone": int,
+    }
+)
+
+GetTimeResponse = TypedDict(
+    "GetTimeResponse",
+    {
+        "Dst": GetTimeDst,
+        "Time": GetTime,
+    }
+)
+
+class _DST_Rule:
```

Comment:
Again can we make this part of the Reolink_timezone class?
I don't like having to many seperate classes